Dockerfiles/cluster-agent/Dockerfile

```diff
@@ -1,24 +1,23 @@
 FROM debian:stretch-slim
-COPY datadog-cluster-agent*_amd64.deb /
 
 LABEL maintainer "Datadog <package@datadoghq.com>"
 
-ENV DOCKER_DD_AGENT=yes \
-    AGENT_VERSION=1:7.0-1 \
-    DD_AGENT_HOME=/opt/datadog-cluster-agent/
+ENV DD_AGENT_HOME=/opt/datadog-cluster-agent/
```

Comment:
what's the purpose of this var?

Reply:
I wanted to expose it for other usages in the code. It's not used so removing